include/eeprom/core/task.hpp

```diff
@@ -187,6 +189,7 @@ class EEPromMessageHandler {
             m.memory_address, iter, (iter + hw_iface.get_eeprom_addr_bytes()));
 
         auto transaction = i2c::messages::Transaction{
+            .message_index = m.message_index,
```

Comment:
So these message indexes for internal messages will have a unique set of errors that the `send_to_can` message will eventually be able to interpret?

Reply:
Passing these into the i2c or spi transactions allows good responses to contain the message ind or we could generate an error message with the message ind